Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -35,4 +53,10 @@ void BatteryStateOfChargeService::addData(const BatteryData& batteryData)
     // This is where you can update your variables
```

Comment:
You can remove these comments now.

Reply:
Also, Q_UNUSED can be removed; it's a macro that causes QT to not throw an error for not using `batteryData`, but we are now!